test/lib/ufe/testRename.py

```diff
@@ -275,6 +275,7 @@ def testRenameRestrictionOtherLayerOpinions(self):
            newName = 'Ball_35_Renamed'
            cmds.rename(newName)
 
+    @unittest.skip("Restriction around renaming external references has been removed.")
```

Comment:
I think we should remove this test, if it's no longer useful.  If we ever need to go back to it, it'll be in git.

Reply:
addressed in 7fafef624444489ef3ddddd200706b9a57ed353c